torch/csrc/jit/python_ir.cpp

```diff
@@ -433,14 +433,27 @@ void initPythonIRBindings(PyObject * module_) {
       switch(t->kind()) {
         case TypeKind::DynamicType:
           return "DynamicType";
+        case TypeKind::TensorType:
+          return "TensorType";
+        case TypeKind::NumberType:
+          return "NumberType";
+        case TypeKind::NoneType:
+          return "NoneType";
         case TypeKind::CompleteTensorType:
           return "CompleteTensorType";
         case TypeKind::TupleType:
           return "TupleType";
-        default:
-          AT_ERROR("unknown type kind");
-          return "";
+        case TypeKind::ListType:
+          return "ListType";
+        case TypeKind::IntType:
+          return "IntType";
+        case TypeKind::FloatType:
+          return "FloatType";
+        case TypeKind::StringType:
+          return "StringType";
```

Comment:
These are used as enum tags in Python (e.g. `foo.type() == "StringType"`), it wouldn't be a good idea to use pretty printing. I edited it so that we will get a warning if you add a enum case.

Reply:
sorry (`foo.type().kind() == "StringType"`)